app/models/enrollment.rb

```diff
@@ -0,0 +1,5 @@
+class Enrollment < ApplicationRecord
+  enum status: { invited: 0, accepted: 1 }
+  belongs_to :user, optional: true
+  belongs_to :event
```

Comment:
I would suggest the following association here because, in your user and event models, you used `attendee` and `attended_event`. You could use the same here as below
* [ ] Enrollment model `belongs_to :attendee, class_name: "User"`
* [ ] Enrollment model `belongs_to :attended_event, class_name: "Event"`